atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -87,7 +113,7 @@ def run(self):
             if not build:
                 break
```

Comment:
`getNextRelease` won't increment the second part - https://pagure.io/koji/blob/master/f/hub/kojihub.py#_3590, for `20.2` it would return `21.2`, which would be adjusted to `21`